Gaming The Syllabus/Assets/Scripts/Combat/CombatManager.cs

```diff
@@ -134,12 +121,37 @@ private static CombatUnit GetRandomLivingEnemy()
 
     }
 
-    public static void StartCombat(CombatUnit player, params CombatUnit[] enemies)
+    public static void StartCombat(CombatUnit player = null, params CombatUnit[] enemies)
```

Comment:
At some point we should consider adding an overload for this method that just takes in the filepaths directly. Seems to me that the deserialization should be the business of the CombatManager and other classes should only be concerned with the XML files existing, not the classes like CombatUnit we use to represent them. I'm open to discussion on this though

Reply:
Also worth thinking about how we want to eventually handle passing in the player. Maybe not at all and load it from the Player component?